adding stomp headers argument into connect method

gordon00 opened the pull request.

gordon00 committed:
adding stomp headers argument into connect method

gordon00 committed:
Make error callback work + selenium integration test

gordon00 committed:
Adding server side websocket closing test

gordon00 committed:
Adding server side websocket closing test

gordon00 committed:
Renaming artifact ID like original project

gordon00 committed:
Old files from when I added sub modules

gordon00 committed:
Checking headers map is not null

kmthinkingrock commented:
I would love to see some traction on this pull request. I have changes to contribute also, and if pull requests are not going to be reviewed and committed, I guess we should fork this gwt-stomp project.

I am not accustomed to github just yet, and also I have not reviewed the changes in this PR. So I am not ready to make a pull request myself, especially since my change includes an API change. I am not sure if it is better to have API change or non-breaking change. Normally I would say non-breaking change but if we have to fork anyway, there is no harm in API change.

Do you have thoughts on this @gordon00 ?

gordon00 commented:
Hi @kmthinkingrock. When I first tested this API, the wrapper was incomplete and it was not able to send stomp headers using connect method. And the error callback was not wrapped. So that is the two major things I added. But I changed the maven structure and added a test module with an embedded tomcat, a Spring sockjs service layer, a GWT test module and some selenium tests to make sure everything works fine.

But since then I have not been using my work (lack of time).
What would you want to add ? Are you going to fork from original API or from my contribution ?
